lib/internal/modules/cjs/loader.js

```diff
@@ -122,9 +122,7 @@ Module._extensions = Object.create(null);
 var modulePaths = [];
 Module.globalPaths = [];
 
-Module.wrap = function(script) {
-  return Module.wrapper[0] + script + Module.wrapper[1];
-};
+Module.wrap = (script) => (Module.wrapper[0] + script + Module.wrapper[1]);
```

Comment:
Is this function exposed anywhere publicly? I wonder if `require('module').wrap` is used anywhere and if so if it's `toString`d or `new`d 

I'd _hope_ not, but if it is this change breaks it)

I'd personally be OK with this change with a CITGM run - if you're not sure what that is don't worry, a project member will start it for you and it doesn't require any involvement on your behalf :)

Reply:
Also, please don't use those parenthesis.

Reply:
So what should I do then?
I think it makes no side effect personally.
And how about the test output?

Reply:
> Also, please don't use those parenthesis.

Without parenthesis will raise lint error

Reply:
> Is this function exposed anywhere publicly? I wonder if `require('module').wrap` is used anywhere and if so if it's `toString`d or `new`d
> 
> I'd _hope_ not, but if it is this change breaks it)
> 
> I'd personally be OK with this change with a CITGM run - if you're not sure what that is don't worry (basically running a bunch of tests), a project member will start it for you and it doesn't require any involvement on your behalf :)

![image](https://user-images.githubusercontent.com/29517764/46421760-6a122b00-c765-11e8-8dfd-010f4d5bc1e5.png)
It's ok for us to call toString() function ,etc

Reply:
> So what should I do then?

I don't mean this to discourage you, but we could just not do anything at all.

Reply:
I'm fine with the parentheses for clarity and I'm curious as to why the request to remove them. /cc @devsnek 

I'm also fine with doing nothing here. Compressing more code into fewer lines can be an anti-pattern in many cases as it hinders readability and maintainability.